LibVLCSharp.Uno/LibVLCSharp.Uno.csproj

```diff
@@ -0,0 +1,44 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="MSBuild.Sdk.Extras/2.0.54">
+  <PropertyGroup>
+    <LangVersion>latest</LangVersion>
```

Comment:
This should be pinned to 8.0.

Reply:
Done in f30631a566c07c0a98001abcb25665e305a869a5